### ISSUE TITLE
Sponsor list doesn't show placename, country and URL of organization

### ISSUE THREAD

**Issue by andreasneumann** [kartoza/projecta#974]
Hi,

In the old, manually edited, sponsors list we had the country and the URL linking directly to the sponsors homepage.

In the new list, the URL links first to Projecta and not directly to the company/organization.

The old list also listed the country of the sponsor, which was quite a useful information. Could we reintroduce the country in the listing and also list the city of the sponsor?

Could you please change this back to the old behavior in addition with listing the city of the sponsor?

Thanks a lot,
Andreas

**Comment by timlinux** [comment]
Is this for the feed onto QGIS.ORG @andreasneumann ?

**Comment by andreasneumann** [comment]
yes - the list published at https://www.qgis.org/en/site/about/sponsorship.html

It always links back to the respective Projecta entry, but the URL should point directly to the sponsor's web page.

Thanks,
Andreas
